marklogic-data-hub-central/ui/src/components/base-entities-facet/base-entities-facet.tsx

```diff
@@ -19,6 +19,8 @@ interface Props {
     setIsAllEntitiesSelected: (isSelected: boolean) => void;
 }
 
+const {MINIMUM_ENTITIES} = exploreSidebar;
```

Comment:
This didn't compile for me. In here and in related-entities-facets I had to change the import line from
import {MINIMUM_ENTITIES} from "../../config/exploreSidebar";
to
import {exploreSidebar} from "../../config/explore.config";

Reply:
Pushed a fix. Another review ?